backend/molgenis-emx2-sql-it/src/test/java/org/molgenis/emx2/sql/TestMgColumns.java

```diff
@@ -0,0 +1,109 @@
+package org.molgenis.emx2.sql;
+
+import static org.junit.Assert.fail;
+
+import org.junit.Assert;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.molgenis.emx2.*;
+import org.molgenis.emx2.Database;
+import org.molgenis.emx2.Row;
+import org.molgenis.emx2.Schema;
+import org.molgenis.emx2.Table;
+
+public class TestMgColumns {
+  private static Schema schema;
+
+  @BeforeClass
+  public static void setUp() {
+    Database database = TestDatabaseFactory.getTestDatabase();
+    schema = database.dropCreateSchema(TestMgColumns.class.getSimpleName());
+  }
+
+  @Test
+  public void testMgDraft() {
+    Table t =
+        schema.create(
+            TableMetadata.table(
+                "MgDraft",
+                Column.column("id").setPkey(),
+                Column.column("required").setRequired(true),
+                Column.column("notrequired")));
+
+    try {
+      t.insert(Row.row("id", 1, "notrequired", "somevalue1"));
+      fail("should fail");
+    } catch (Exception e) {
+      // ok
+    }
+
+    try {
+      t.insert(Row.row("id", 1, "notrequired", "somevalue2").setDraft(true));
+    } catch (Exception e) {
+      fail("should succeed");
+    }
+
+    // verify
+    Assert.assertEquals(1, t.retrieveRows().size());
+    Assert.assertEquals("somevalue2", t.retrieveRows().get(0).getString("notrequired"));
```

Comment:
test is hard to read / reason about what is being testes , suggest splitting test into named test cases and use ```@Test(expected = ...``` combined with throws to simplify control flow

Reply:
> 95% of code changes seem to be swapping out .* imports and class level imports with explicitly referencing the class when used, would have been a small PR if these changes were splint into separate PR

Correct. Only thing I did was
- move the tests from molgenis-emx2-sql to molgenis-emx2-sqlt-it
- changed initialisation of the test Database (if applicable), making it configurable for each test suite instead of only central